server/users/users.controller.ts

```diff
@@ -83,16 +71,17 @@ export class UsersController {
     }
 
     @Get("profile")
-    public async profile(@Req() req: Request, @Res() res: Response) {
+    public async profile(@Req() req, @Res() res: Response) {
```

Comment:
any reason for removing the type hint here?

Reply:
The typing was not recognizing the req.user._id

Reply:
What do you think about adding a new type? Maybe something like:
```
type BaseRequest = Request & { user: { _id: string }}
```